README.md

```diff
@@ -30,16 +37,22 @@ $ dbt seed
 
 ### Running Instructions:
 
-Run fal flow
+1. Run `dbt run`
 
 ```bash
-$ fal flow run --experimental-models
-## runs the whole graph
+$ dbt run
+## Runs the whole graph including the Python models
 ```
 
-Alternatively run fal flow with [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators)
+2. Run `fal flow` to execute the full graph including Python scripts. You can use the dbt [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators) and [much more](https://docs.fal.ai/).
 
 ```bash
-$ fal flow run --experimental-models --select order_detailed_cluster.py+
-## runs order_detailed_cluster.py and cluster_stats.sql
+$ fal flow run
+## Runs dbt run and the associated scripts, in this case a Slack notification is triggered
 ```
```

Comment:
With `fal flow run` the user doesn't need to do `dbt run`, right? We can make a suggestion that they can use dbt but then the post-hooks won't run.